L2.py

```diff
@@ -0,0 +1,61 @@
+from random import *
+from copy import *
+import time
+
+def trzy_pierwsze_wartosci(lista, wp):
+    indeksy = []
+    wartosci = []
+    licznik = 0
+    for i in range(len(lista)):
+        if lista[i] > wp:
+            indeksy.append(i)
+            wartosci.append(lista[i])
+            licznik += 1
+        if licznik == 3:
+            break
+
+    return indeksy, wartosci
+
+def sortowanie_1(lista,wp):
+    index, wartosci = trzy_pierwsze_wartosci(lista, wp)
```

Comment:
trzy_pierwsze_wartości to mało uniwersalna nazwa funkcji, wyobraź sobie, że  w następnym kroku rozwój Twojego programu polega na wyciągnięciu N pierwszych wartości. Gdybyś dobrze napisał funkcję trzeba byłoby tylko wprowadzić nową zmienną jako argument wejściowy, a tu nazwę funkcji (co za tym idzie wszystkie jej wywołania w całym projekcie, który potencjalnie może być duży).